### PR TITLE
Make better URL & small fixes

### PR DESCRIPTION
**Pull request by joolswood**
A few small fixes, and linking up to parent page.

### CODE REVIEW

File: content/governing-content/content-lifecycle/signing-off-content/sign-off-template/template.md [hunk 1]
```diff
@@ -12,10 +12,11 @@ We’re asking for your sign off to publish:
 [insert staging URL if appropriate].
 
 Below is a checklist of what we’ve already validated.
+
 We’d specifically like you to review this content for the following:
 
-organisational risk
-specific concerns such as [insert as appropriate]
+• organisational risk
```
Comment (marksantoso) [inline content/governing-content/content-lifecycle/signing-off-content/sign-off-template/template.md]: are these the correct bullet @joolswood

Comment (joolswood) [reply content/governing-content/content-lifecycle/signing-off-content/sign-off-template/template.md]: Can't seem to get markdown versions to show up?

Comment (marksantoso) [reply content/governing-content/content-lifecycle/signing-off-content/sign-off-template/template.md]: did you try with a space between the - and text?

Comment (joolswood) [reply content/governing-content/content-lifecycle/signing-off-content/sign-off-template/template.md]: Yes

Comment (joolswood) [reply content/governing-content/content-lifecycle/signing-off-content/sign-off-template/template.md]: They just render as a line break

**Pull request merged**